src/components/HogTile.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import HogDetails from "./HogDetails";
+
+class HogTile extends React.Component {
+  state = {
+    clicked: false
+  };
+
+  getImage = hogName => {
+    let formattedName = hogName
+      .split(" ")
+      .join("_")
+      .toLowerCase();
+    let pigPics = require(`../hog-imgs/${formattedName}.jpg`);
+    return pigPics;
+  };
+
+  handleDetailsClick = () => {
+    //when clicked, change clicked to true
+    this.setState({ clicked: !this.state.clicked });
+  };
+
+  render() {
+    const { name, specialty } = this.props.hog;
+    return (
+      <div className="ui card eight wide column pigTile">
+        <div className="image">
+          <img src={this.getImage(name)} alt="hogPic" />
+        </div>
+        <div className="content">
+          <h3 className="header">{name}</h3>
+          <div className="description">Specialty: {specialty}</div>
+        </div>
+        <div className="extra content">
+          {this.state.clicked ? <HogDetails hog={this.props.hog} /> : null}
+
+          <button className="ui button" onClick={this.handleDetailsClick}>
+            {this.state.clicked ? "Less Info" : "More Info"}
+          </button>
+          <button
+            className="ui button"
+            onClick={() => this.props.handleBanishedClick(this.props.hog)}>
+            Hide Me{" "}
+            <span role="img" aria-label="snout">
+              🐽
```

Comment:
![image](https://user-images.githubusercontent.com/19658749/42537910-403d7bac-8464-11e8-8ded-02b4c47f5819.png)

Reply:
Did you want me to do something here?

![bitmoji](https://render.bitstrips.com/v2/cpanel/f4c91480-fbcf-41d6-8afe-2c9b58d18798-1bf2df23-0b61-48ca-bcdb-8f610cdae042-v1.png?transparent=1&palette=1&width=246)

Reply:
no lol i like the emoji